zebrad/src/commands/tests.rs

```diff
@@ -16,7 +16,7 @@ fn args_with_subcommand_pass_through() {
         (true, false, false, vec!["zebrad", "--help"]),
         (false, true, false, vec!["zebrad", "start"]),
         (false, true, true, vec!["zebrad", "-v", "start"]),
-        (false, true, false, vec!["zebrad", "warn"]),
+        (false, true, false, vec!["zebrad", "--filters", "warn"]),
```

Comment:
Can you also add a test case for detecting an invalid subcommand?
I'd also like to check that `zebrad start debug` works without `--filters`, like it does in v1.0.1.

Reply:
> https://github.com/ZcashFoundation/zebra/pull/7056/commits/26a7295b53ef8f3a20764dbb83de7d4e0999f52b

Added in https://github.com/ZcashFoundation/zebra/pull/7056/commits/26a7295b53ef8f3a20764dbb83de7d4e0999f52b.

> I'd also like to check that zebrad start debug works without --filters, like it does in v1.0.1.

This should be checked by the second-last test case, `zebrad start warn`.